udata_front/settings.py

```diff
@@ -77,3 +77,6 @@
 
 # Metrics API
 METRICS_API = None
+
+# recherche-entreprises.api.gouv.fr
+SEARCH_SIREN = 'https://recherche-entreprises.api.gouv.fr/search'
```

Comment:
call this setting, the meta and config `SEARCH_SIRET_URL` to better describe its usage